src/bindings/bot-runtime.ts

```diff
@@ -0,0 +1,60 @@
+import { Context } from "probot";
+import { BotConfig } from "../types";
+import { createAdapters } from "../adapters";
+import { Logs } from "../adapters/supabase";
+
+class Runtime {
```

Comment:
this will still cause the issue described in #732 because there's only one instance of Runtime and multiple requests will override each other. this is fine for adapters and logger because they should only be initialized once, but event context and config should not be global/singleton

Reply:
I'm working on your issue now.